### PR TITLE
Feat(UI): Added webpack bundle analyser for dev and prod

### PR DESCRIPTION
**Pull request by Anurag5086**
Solves #421

### CODE REVIEW

File: frontend/webpack.config.js [hunk 1]
```diff
@@ -73,4 +75,8 @@ const config = {
 	},
 };
 
+if (process.env.BUNDLE_ANALYSER === 'true') {
```
Comment (pal-sig) [inline frontend/webpack.config.js:78]: const plugin = // all required Plugins 

if(devEnv){
plugin.push(bundleAnalyser)
}

then use plugins

**Review by pal-sig — CHANGES_REQUESTED** [review]

File: frontend/webpack.config.js (hunk 1 above)
Comment (Anurag5086) [reply frontend/webpack.config.js:78]: devEnv refers to the NODE_ENV variable? As we have separate webpack config files for prod and dev. So how could this condition be used?

Comment (pal-sig) [reply frontend/webpack.config.js:78]: sorry my bad here i was referring `devEnv` to  

```
process.env.BUNDLE_ANALYSER === 'true'
```

Comment (Anurag5086) [reply frontend/webpack.config.js:78]: Okay alright!

**Review by pal-sig — APPROVED** [review]

**Pull request merged**